File: 0x0A-argc_argv/2-args.c
#include "main.h"

/**
* main - Print all arguments
* @argc: Argument count
* @argv: Argument vector
* Return: 0 (Success)
*/
int main(int argc, char **argv)
{
	for (int i = 0; i < argc; i++)
	{
		printf("%s\n", argv[i]);
	}
	return (0);
}

File: 0x02-python-import_modules/2-args.py
#!/usr/bin/python3
import sys

def print_args():
    argc = len(sys.argv) - 1  # Définition de argc dans la fonction

    if argc == 0:
        print("0 arguments.")
    elif argc == 1:
        print("1 argument:")
    else:
        print("{} arguments:".format(argc))

    for i in range(1, argc + 1):
        print("{}: {}".format(i, sys.argv[i]))

if __name__ == "__main__":
    print_args()
